conf/machine/raspberrypi5.conf

```diff
@@ -0,0 +1,30 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi 5 Development Board (64bit)
+#@DESCRIPTION: Machine configuration for the RaspberryPi 5 in 64 bits mode
```

Comment:
nitpick:  I wonder how relevant it is to mention 64-bit since it is 64-bit **only**.

Reply:
I'll leave it as such for now for clarity, as we have a mix of 32/64 in the other machines.